README.md

```diff
@@ -71,6 +71,7 @@ Considering the balance between correctness and performance, `sse2neon` recogniz
 * `SSE2NEON_PRECISE_DP`: Enable precise implementation of `_mm_dp_pd`. When the conditional bit is not set, the corresponding multiplication would not be executed.
 
 The above are turned off by default, and you should define the corresponding macro(s) as `1` before including `sse2neon.h` if you need the precise implementations.
+The configuration macros are in the `config.h`.
```

Comment:
Don't do that. At present, SSE2NEON follows the single-file style as [stb](https://github.com/nothings/stb) does. Only one header file is deployed and used. The extra `config.h` might cause unexpected path matching order.

Reply:
I will find another way to handle the testing without extra header file.

Reply:
I reverse it and add the macro in `tests/common.h` when the testing suite does not include `sse2neon.h`.